app/audit/GetAFTDetailsAuditService.scala

```diff
@@ -37,6 +37,16 @@ class GetAFTDetailsAuditService @Inject()(auditService: AuditService) {
       sendEvent(pstr, startDate, error.responseCode, None)
   }
 
```

Comment:
minor comment , I would suggest assigning the return type to 
type Event = PartialFunction[Try[Option[JsValue]], Unit]

Reply:
I meant to change the signature of the function from 
def sendOptionAFTDetailsAuditEvent(pstr: String, startDate: String)
                              (implicit ec: ExecutionContext, request: RequestHeader): PartialFunction[Try[Option[JsValue]], Unit]

to 

def sendOptionAFTDetailsAuditEvent(pstr: String, startDate: String)
                              (implicit ec: ExecutionContext, request: RequestHeader): Event 

by using type.

Reply:
Done